types/sinon/ts3.1/sinon-tests.ts

```diff
@@ -160,11 +160,21 @@ function testClock() {
     clock.cancelAnimationFrame(animTimer);
 
     clock.nextTick(fn);
+
     clock.tick(1);
     clock.tick('00:10');
+    clock.tickAsync(500).then((val: number) => val.toExponential());
```

Comment:
i'd leave the types out here, we're trying to ensure the correct one is inferred really

Reply:
change pushed up, will squash again once things are fully approved